shared/chat/conversation/input-area/normal/container.js

```diff
@@ -63,9 +63,11 @@ const mapStateToProps = (state: TypedState, {conversationIDKey}) => {
   const _you = state.config.username || ''
   const pendingWaiting = state.chat2.pendingSelected && state.chat2.pendingStatus === 'waiting'
 
-  const injectedInputMessage = _editingMessage || _quotingMessage || null
-  const injectedInput =
-    injectedInputMessage && injectedInputMessage.type === 'text' && injectedInputMessage.text.stringValue()
+  const injectedInputMessage: ?Types.Message = _editingMessage || _quotingMessage || null
+  const injectedInput: ?string =
+    injectedInputMessage && injectedInputMessage.type === 'text'
+      ? injectedInputMessage.text.stringValue()
+      : null
```

Comment:
would It be a problem to set the `''` here instead? If so no big deal, I just thought since this is the source of the value it'd make sense to guarantee stringy-ness here.

Reply:
Sure, done.